jobrunner/queries.py

```diff
@@ -3,10 +3,15 @@
 from itertools import groupby
 from operator import attrgetter
 
+from opentelemetry import trace
+
 from jobrunner.lib.database import find_all, find_one, find_where, upsert
 from jobrunner.models import Flag, Job
 
 
+tracer = trace.get_tracer("db")
```

Comment:
I'm guessing this belongs in the next commit as it's not used here (not that it really matters)?